### ISSUE TITLE
CTOV loot table fails to load and prevents lootr chests from filling.

### ISSUE THREAD

**Issue by dbravo4** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#38]
**Describe the bug**
The CTOV loot table fails to load which prevents Lootr chests from filling. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the server with CTOV and Lootr installed.
2. Check logs for loot table error (as shown in logs section)
3. Lootr chests will fail to fill (also shown in logs section)

**Expected behaviour**
The loot table should successfully load so that Lootr chests can successfully fill.

**Screenshots**
N/A

**Minecraft Version**
1.19.2
**CTOV Version**
3.2.1
**Log**

`[main/WARN] [net.minecraft.world.level.storage.loot.LootTables/]: Found validation problem in {ctov:chests/village/village_library}.pools[0].entries[0]: Unknown loot table called ctov:village/common/library`

```
[Server thread/ERROR] [noobanidus.mods.lootr.api.LootrAPI/]: Unable to fill loot chest in ResourceKey[minecraft:dimension / minecraft:overworld] at BlockPos{x=943, y=120, z=421} as the loot table 'ctov:village/mesa_fortified/generic' couldn't be resolved! Please search the loot table in `latest.log` to see if there are errors in loading.
[Server thread/ERROR] [noobanidus.mods.lootr.api.LootrAPI/]: Unable to fill loot chest in ResourceKey[minecraft:dimension / minecraft:overworld] at BlockPos{x=952, y=120, z=424} as the loot table 'ctov:village/mesa_fortified/generic' couldn't be resolved! Please search the loot table in `latest.log` to see if there are errors in loading.
```
**Additional context**
This error appeared after updating All the Mods 8 to 1.0.18 which updated CTOV to 3.2.1 (I am not sure what the past version was)

**Comment by ChoiceTheorem** [comment]
They are some error, I'll look into it but I won't rule-out mod-mod interaction.

**Comment by ChoiceTheorem** [comment]
The same behaviour is spotted for smiths.

**Comment by DRACTION** [comment]
Same problem on version 2.9.4 for 1.18.2
<details>
<summary>Log</summary>
```
[22:16:47] [Server thread/ERROR] [no.mo.lo.ap.LootrAPI/]: Unable to fill loot chest in ResourceKey[minecraft:dimension / minecraft:overworld] at BlockPos{x=-253, y=105, z=133} as the loot table 'ctovt:chests/village/village_smith' couldn't be resolved! Please search the loot table in `latest.log` to see if there are errors in loading.
[22:17:31] [Server thread/ERROR] [no.mo.lo.ap.LootrAPI/]: Unable to fill loot chest in ResourceKey[minecraft:dimension / minecraft:overworld] at BlockPos{x=-266, y=104, z=129} as the loot table 'ctov:chests/village/village_library' couldn't be resolved! Please search the loot table in `latest.log` to see if there are errors in loading.
```
</details>

**Comment by ChoiceTheorem** [comment]
Will likely be fixed only in 1.19 series
